README.md

```diff
@@ -85,4 +85,8 @@ Contributions welcome!
 - เจ้าจะทำอะไรก็เรื่องของเค้า
 - จ่ายภาษีหรือเปล่า
 - ทำไมไม่เคารพความเห็นต่าง
+- เป็น นร ก็กลับไปตั้งใจเรียนหนังสือ
+- เด็กๆเอาแต่เล่นโซเชียล fake news ทั้งนั้น
+- รัฐบาลมีผลงานเยอะแยะ แค่ข่าวไม่ออกเท่านั้นแหละ
+- เป็นลูกพระเจ้าตากรึเปล่า
```

Comment:
I think `ลูกพระเจ้าตาก` is being used more on pro-democracy side, consider removing just this line